modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/PaginationBar.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayPaginationBarWithBasicItems} from '@clayui/pagination-bar';
+import React, {useState} from 'react';
+
+export default function PaginationBar({
+	activeDelta,
+	activePage,
+	componentId: _componentId,
```

Comment:
You are aliasing those properties with `_` prefix but it's unused

Reply:
exactly, that's why I'm doing it, to filter them out

Reply:
Great! I understood! It's just to omit some properties of `...otherProps` to being passed to the main component like ClayButton, ClayCheckbox :)